### PR TITLE
[webkitscmpy] Stop killing finished processes

### PR DESCRIPTION
**Pull request by JonWBedard**
#### 85bc38c88026101e1c62f5d063e6809529dfbb85
<pre>
[webkitscmpy] Stop killing finished processes
<a href="https://bugs.webkit.org/show_bug.cgi?id=235427">https://bugs.webkit.org/show_bug.cgi?id=235427</a>
&lt;rdar://problem/87887233 &gt;

Rubber-stamped by Aakash Jain.

* Tools/Scripts/libraries/webkitscmpy/setup.py: Bump version.
* Tools/Scripts/libraries/webkitscmpy/webkitscmpy/__init__.py: Ditto.
* Tools/Scripts/libraries/webkitscmpy/webkitscmpy/local/git.py:
(Git.Cache.populate): Check if process has finished before killing it.
(Git.commits): Ditto.
* Tools/Scripts/libraries/webkitscmpy/webkitscmpy/local/svn.py:
(Svn.Cache.populate): Check if process has finished before killing it.
(Svn.commits): Ditto.

Canonical link: <a href="https://commits.webkit.org/246268@main">https://commits.webkit.org/246268@main</a>
git-svn-id: <a href="https://svn.webkit.org/repository/webkit/trunk@288366">https://svn.webkit.org/repository/webkit/trunk@288366</a> 268f45cc-cd09-0410-ab3c-d52691b4dbfc
</pre>

### CODE REVIEW

File: Tools/ChangeLog [hunk 1]
```diff
@@ -1,3 +1,18 @@
+2022-01-21  Jonathan Bedard  <jbedard@apple.com>
+
+        [webkitscmpy] Stop killing finished processes
+        https://bugs.webkit.org/show_bug.cgi?id=235427
+        <rdar://problem/87887233>
+
+        Reviewed by NOBODY (OOPS!).
+
+        * Scripts/libraries/webkitscmpy/webkitscmpy/local/git.py:
+        (Git.Cache.populate): Check if process has finished before killing it.
```
Comment (aj062) [inline Tools/ChangeLog]: Does it solve the problem of "No such process"?

**Review by aj062 — APPROVED** [review]
rs=me

File: Tools/ChangeLog (hunk 1 above)
Comment (JonWBedard) [reply Tools/ChangeLog]: It _should_, but it's not a problem I've been able to reproduce. It's the only way I can see us getting into that codepath, though. It's likely to be a pretty rare and race-y failure as is.

**Comment by JonWBedard** [comment]
Landed https://commits.webkit.org/246268@main (85bc38c88026101e1c62f5d063e6809529dfbb85)!

**Pull request merged**